src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/StreamingClientManager.java

```diff
@@ -136,6 +136,15 @@ public KcStreamingIngestClient getValidClient(int taskId) {
     return client;
   }
 
+  /**
+   * Gets the number of clients created
+   *
+   * @return the number of clients created
+   */
+  public int getClientCount() {
+    return this.clientId + 1; // clientid starts at 0, so off by one
```

Comment:
more of `starts at -1`?

Reply:
Updated comment to be more clear, thanks for pointing out the confusion here